pom.xml

```diff
@@ -49,6 +49,7 @@
          https://github.com/confluentinc/common (see common's pom.xml for kafka.version).
     -->
     <properties>
+	<dropwizard.version>4.1.12.1</dropwizard.version>
```

Comment:
Move this line -- hope this will avoid merge conflict down the line using `pint-merge`.